chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -268,6 +269,7 @@ export class TelemetryConstants {
             case TelemetryEvent.PreChatSurveyStartChatMethodFailed:
             case TelemetryEvent.HeaderCloseButtonClicked:
             case TelemetryEvent.HeaderMinimizeButtonClicked:
+            case TelemetryEvent.NotificationCloseChatButtonClicked:
```

Comment:
Do we have same telemetry added if the Notification dismiss button is clicked ?

Reply:
Added notification dismiss button telemetry